cmd/breaking/find.go

```diff
@@ -0,0 +1,76 @@
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"os/exec"
+	"regexp"
+
+	"github.com/pkg/errors"
```

Comment:
error traceability - we're switching over to using pkg errors almost everywhere with tendermint